### DISCUSSION TITLE
AT3 board - no buttons

### DISCUSSION THREAD

**Discussion by pkarza** [keirf/flashfloppy#950] [Q&A]
I bought a sfrkc30.at3 board.  I was able to add the rotary switch to the interface and added the 9.1 display.  What IS strange is that there  are NO push buttons on the front.  Is that a problem?  should I solder some there?  

Also - I am somewhat confused as to how to use the rotary.  I can put .adf files (Amiga) on the USB - I put them all in the root directory - and I can rotate to the file (they display OK) - BUT how do you load the file?? Is there a readme on how to use this.  I know I will figure it out - my biggest question was about the missing buttons.

Thanks for help, in advance.

**Comment by keirf** [comment]
9.1 display: is that an OLED display?

You should be able to select by pressing the rotary encoder, but in fact the default is to auto-select after two seconds inactivity anyway. Connection of a rotary encoder is covered here: https://github.com/keirf/flashfloppy/wiki/Hardware-Mods#rotary-encoder

See specifically the subsection on "KC30 Rotary Header", which your PCB model has, and is what you should connect to.

From the above link you can also find the rest of the documentation wiki, including the FF.CFG file which allows you to modify FlashFloppy behaviour, should you need to (many users don't need to).

**Reply by pkarza** [reply]
Thanks, Keirf.

One last question - is it better to use this as the internal and put the mechanical drive external on a 500?  What would be your preference.

**Reply by keirf** [reply]
That's a tough one. Without a 3d printed sled or similar, the Gotek is quite hard to neatly fit inside an A500. So some people simply stick them outside, but a lot of games want to load from DF0. Actually if you really care to use both mechanical drive and Gotek for loading games you're going to need a DF0 switcher. And then it's easier just to keep the Gotek as the external drive.

There are clever solutions that support both mechanical drive and Gotek inside the A500. Such as https://digitalretrobay.co.uk/digital-godrive-for-commodore-amiga-500-500/ -- these are custom Gotek designs, basically.
